recipes/nuraft/all/conanfile.py

```diff
@@ -0,0 +1,78 @@
+from os.path import join
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import copy, get, apply_conandata_patches
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conan.tools.build import check_min_cppstd
+
+required_conan_version = ">=1.50.0"
+
+class NuRaftConan(ConanFile):
+    name = "nuraft"
+    homepage = "https://github.corp.ebay.com/sds/NuRaft"
+    description = """Cornerstone based RAFT library."""
+    topics = ("raft")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Apache-2.0"
+
+    settings = "os", "compiler", "arch", "build_type"
+
+    options = {
+        "asio": ["boost", "standalone"],
+        "shared": ['True', 'False'],
+        "fPIC": ['True', 'False'],
+    }
+    default_options = {
+        "asio": "boost",
+        "shared": False,
+        "fPIC": True,
+    }
+
+    def export_sources(self):
+        for p in self.conan_data.get("patches", {}).get(self.version, []):
+            copy(self, p["patch_file"], self.recipe_folder, self.export_sources_folder)
+
+    def requirements(self):
+        if self.options.asio == "boost":
+            self.requires("boost/1.79.0")
```

Comment:
```suggestion
            self.requires("boost/1.80.0")
```
We have Boost 1.80.0 available on Conan Center.

Reply:
Is it not better to target a lower MINOR since it won’t require recompile should someone want 1.80.0 as opposed to the reverse which *would*?

Reply:
soon or later, all recipes will use 1.80.0, then your recipe will need to be updated too. Doing it now, it are just doing what should be done later.

Reply:
In ConanCenter we always try to stay on latest to avoid conflicts... two recipes using boost should work together (hopefully)